scripts/irods/test/test_imeta_set.py

```diff
@@ -355,6 +355,26 @@ def test_imeta_handles_trailing_slashes_on_collections__issue_4559(self):
         self.admin.assert_icommand(['imeta', 'ls', '-C', os.path.join(collection) + '/'],  'STDOUT', ['None'])
         self.admin.assert_icommand(['imeta', 'ls', '-C', os.path.join(collection) + '//'], 'STDOUT', ['None'])
 
+    def test_imeta_handles_setting_separate_collections_to_same_avu_then_to_empty_avu__issue_4063(self):
+        # Generate collections for test
+        collections = ['games', 'super_secret_documents']
+        collections = [os.path.join(self.admin.session_collection, col) for col in collections]
+
+        # Create collections
+        for collection in collections:
+            self.admin.assert_icommand(['imkdir', collection])
+
+        # Set the same AVUs for the collections
+        for collection in collections:
+            self.admin.assert_icommand(['imeta', 'set', '-C', collection, 'is_cool', 'yes'])
+
+        # Clear Value of second collection
+        out, err, ec = self.admin.run_icommand(['imeta', 'set', '-C', collections[1], 'is_cool', "''"])
+
```

Comment:
Please also assert that the AVUs on both collections are what you want them to be.